tests/Fixer/FunctionNotation/VoidReturnFixerTest.php

```diff
@@ -55,6 +55,20 @@ public function provideFixCases()
                  */
                 function foo($param) {}
             '],
+            ['<?php
+                /**
+                 * @return void|array
```

Comment:
I wanted to test `void|something` to make sure my new conditions were working.  Guess it should have been `something|void`.  I can remove if you want.  You can do it, you just cannot type hint on it (EG: conditionally return an array or nothing).